src/huggingface_hub/hf_api.py

```diff
@@ -7518,6 +7519,7 @@ def create_inference_endpoint(
             ...         },
             ...         "url": "ghcr.io/huggingface/text-generation-inference:1.1.0",
             ...     },
+            ...    secrets={"HF_TOKEN": "your-token},
```

Comment:
```suggestion
            ...    secrets={"MY_SECRET_KEY": "secret_value"},
```

(to avoid confusion with user tokens)